client/src/App.js

```diff
@@ -1,25 +1,40 @@
 import React from "react";
-import logo from "./logo.svg";
-import "./App.css";
+import { createGlobalStyle } from "styled-components/macro";
+
+import { Badge } from "./stories/Badge";
+import { HeaderMain } from "./stories/HeaderMain";
+
```

Comment:
Never import your stories from your app or components.
The stories are only for Storybook.

Reply:
so how would you dot it? All our components are based in `./src/stories/`

Reply:
Yes. Stories are use cases for your components and demonstrate how you could use them.